### ISSUE TITLE
A trailing comma is not permitted after the rest element: const {shouldUpdated, ...props, } = Props;

### ISSUE THREAD

**Issue by SUpermin6u** [happypancake/react-native-scrollable-tab-view#932]
Please fix this ok?
```
error: bundling failed: SyntaxError: /Users/superman/Projects/exbuzz-app-react-native/node_modules/react-native-scrollable-tab-view/SceneComponent.js: A trailing comma is not permitted after the rest element (9:32)

   7 | 
   8 | const SceneComponent = (Props) => {
>  9 |   const {shouldUpdated, ...props, } = Props;
     |                                 ^
  10 |   return <View {...props}>
  11 |       <StaticContainer shouldUpdate={shouldUpdated}>
```

**Comment by ammoradi** [comment]
#928

**Comment by ptomasroos** [comment]
This has now been published, truly sorry for the delay on this: 0.9.0
